core/src/lib/card/card.component.ts

```diff
@@ -13,9 +28,42 @@ export type RoleType = 'product-showcase';
     class: 'garuda-card',
   },
 })
-export class CardComponent {
+export class CardComponent implements AfterViewInit, OnDestroy {
   cardRole = input.required<RoleType>();
+  @Input() feature?: FeatureData;
```

Comment:
This shoudld optimally be input signals (feature = input<FeatureData>(...), the some for outputs.